File: 프로그래머스/unrated/181927. 마지막 두 원소/마지막 두 원소.java
class Solution {
    public int[] solution(int[] num_list) {
        int last = num_list[num_list.length - 1];
		int lastMinusOne = num_list[num_list.length - 2];
		int[] answer = new int[num_list.length + 1];
		System.arraycopy(num_list, 0, answer, 0, num_list.length);
		answer[num_list.length] = last > lastMinusOne ? last - lastMinusOne : last * 2;
		return answer;
    }
}

File: 프로그래머스/unrated/181927. 마지막 두 원소/마지막 두 원소.kt
class Solution {
    fun solution(num_list: IntArray): IntArray {
        val last = num_list[num_list.size - 1]
        val lastMinusOne = num_list[num_list.size - 2]
        return num_list + if (last > lastMinusOne) last - lastMinusOne else last * 2
    }
}